src/python/pants/core/goals/fmt.py

```diff
@@ -241,11 +249,19 @@ async def fmt(
         merged_formatted_digest = await Get[Digest](MergeDigests(changed_digests))
         workspace.materialize_directory(DirectoryToMaterialize(merged_formatted_digest))
 
-    for result in individual_results:
+    sorted_results = sorted(individual_results, key=lambda res: res.formatter_name)
+    for result in sorted_results:
+        console.print_stderr(
+            f"{console.green('✓')} {result.formatter_name} made no changes."
+            if not result.did_change
+            else f"{console.red('𐄂')} {result.formatter_name} made changes."
+        )
         if result.stdout:
-            console.print_stdout(result.stdout)
+            console.print_stderr(result.stdout)
```

Comment:
Moving forward (and if we think that it generalizes) we should consider whether we want to indent tool outputs by a small amount (ie, nowhere near the unbounded indentation of v1). So I think we'd have exactly two levels of indentation: the workunit containers would be un-indented, and tool outputs would be indented.